### DISCUSSION TITLE
current state of collaboration between WikiPathways and Reactome

### DISCUSSION THREAD

**Discussion by alexmadsen1** [wikipathways/wikipathways-help#128] [Q&A] [answered]
I'm just wondering about the current state of collaboration between WikiPathways and Reactome. I would like to know whether this collaboration is ongoing.

Specifically, I am working on pathway WP5420 related to B12/Cobalamin metabolism and would like to explore the possibility of porting this information to Reactome. Please advise if there is a good point of contact for discussing this further or guide me through the process. Do you know if we have a point of contact to help make an introduction and facilitate?

Alex Madsen

**Comment by khanspers** [comment, marked as answer]
Hi @alexmadsen1. Our current involvement with Reactome is limited to internal conversion of Reactome content to gpml, which are stored in a separate repo (https://github.com/wikipathways/reactome-gpml). There is no active effort to include curation data from WikiPathways into Reactome pathways at Reactome.
